### PR TITLE
Add redirects for wiki.rtapp.tk and wiki.recaptime.tk as per T9210

### PR DESCRIPTION
**Pull request by ajhalili2006**
ps: I'm not sure if `additional_domains` is fine in case I would add another domain to be aliased in the future.

### CODE REVIEW

File: redirects.yaml [hunk 1]
```diff
@@ -506,6 +506,12 @@ recaptimewiki:
   ca: 'Sectigo'
   hsts: 'strict'
   disable_event: true
+wikirtapptk:
```
Comment (Universal-Omega) [inline redirects.yaml:509]: This isn't an *.miraheze.org redirect, so it should be moved above to be under irishfreemaninfo.com.

**Review by Universal-Omega — CHANGES_REQUESTED** [review]

File: redirects.yaml (hunk 1 above)
Comment (ajhalili2006) [reply redirects.yaml:509]: Thanks for the note, editing right now.

File: redirects.yaml [hunk 2]
```diff
@@ -328,6 +328,12 @@ wwwirishfreemaninfocom:
   sslname: 'irishfreemaninfo.com'
   ca: 'LetsEncrypt'
   disable_event: false
+wikirtapptk:
+  url: 'wiki.rtapp.tk'
+  additional_domain: 'wiki.recaptime.tk'
+  redirect: 'wiki.recaptime.eu.org'
+  ca: "LetsEncrypt"
+  disable_event: true
```
Comment (Universal-Omega) [inline redirects.yaml:336]:
```suggestion
  disable_event: false
```

File: redirects.yaml [hunk 3]
```diff
@@ -328,6 +328,12 @@ wwwirishfreemaninfocom:
   sslname: 'irishfreemaninfo.com'
   ca: 'LetsEncrypt'
   disable_event: false
+wikirtapptk:
+  url: 'wiki.rtapp.tk'
+  additional_domain: 'wiki.recaptime.tk'
```
Comment (Universal-Omega) [inline redirects.yaml:333]: What's this for? This is a very rare thing to need to use. It's currently only used for like 2 other domains which need wildcards I think. Also we need sslname set for this redirect I think.

**Review by Universal-Omega — CHANGES_REQUESTED** [review]

**Comment by MacFan4000** [comment]
I don't think LE does multi-domain certs, so will have to be separate ones

File: redirects.yaml [hunk 4]
```diff
@@ -328,6 +328,18 @@ wwwirishfreemaninfocom:
   sslname: 'irishfreemaninfo.com'
   ca: 'LetsEncrypt'
   disable_event: false
+wikirtapptk:
+  url: 'wiki.rtapp.tk'
+  sslname: 'wiki.recaptime.eu.org'
+  redirect: 'wiki.recaptime.eu.org'
+  ca: "LetsEncrypt"
```
Comment (Universal-Omega) [inline redirects.yaml:335]: Minor, but for consistency:
```suggestion
  ca: 'LetsEncrypt'
```

**Comment by ajhalili2006** [comment]
After the review, can you squash and merge while leaving the full link to the Phabricator Maniphest task?

File: redirects.yaml [hunk 5]
```diff
@@ -328,6 +328,18 @@ wwwirishfreemaninfocom:
...
+  sslname: 'wiki.recaptime.eu.org'
+  redirect: 'wiki.recaptime.eu.org'
+  ca: 'LetsEncrypt'
+  disable_event: false
+wikirecaptimetk:
+  url: 'wiki.recaptime.tk'
+  sslname: 'wiki.recaptime.eu.org'
+  redirect: 'wiki.recaptime.eu.org'
+  ca: 'LetsEncrypt'
+  disable_event: false
```
Comment (Universal-Omega) [inline redirects.yaml:342]:
```suggestion
wikirtapptk:
  url: 'wiki.rtapp.tk'
  redirect: 'wiki.recaptime.eu.org'
  sslname: 'wiki.rtapp.tk'
  ca: 'LetsEncrypt'
  disable_event: false
wikirecaptimetk:
  url: 'wiki.recaptime.tk'
  redirect: 'wiki.recaptime.eu.org'
  sslname: 'wiki.recaptime.tk'
  ca: 'LetsEncrypt'
  disable_event: false
  ```

**Review by Universal-Omega — CHANGES_REQUESTED** [review]
So you want to redirect

**Comment by Universal-Omega** [comment]
> I don't think LE does multi-domain certs, so will have to be separate ones

I opened #518 for that.

**Pull request merged**